Source/Renderer/DrawCommand.js

```diff
@@ -32,6 +32,17 @@ define([
         this.boundingVolume = options.boundingVolume;
 
         /**
+         * The oriented bounding box of the geometry in world space. If this is defined, it is used instead of
+         * {@link DrawCommand#boundingVolume} for plane intersection testing.
+         *
+         * @type {OrientedBoundingBox}
+         * @default undefined
+         *
+         * @see DrawCommand#debugShowBoundingVolume
+         */
+        this.boundingOBB = options.boundingOBB;
```

Comment:
In the debug code which draws the bounding volumes for the inspector, there is different code for spheres and boxes. Is there a way to check there which it is if they're combined? That would be bad design in general, but seems okay for debug code (?)